src/Json/JsonFormatter.php

```
@@ -2,28 +2,63 @@
+     * @param int $indent The space count for indent.
+     * @param bool $formatJson Check if the JSON must be formatted.
+     *
+     * @psalm-param string[] $arrayKeys The list of keys to be retained with an array representation if they are empty.
+     *
+     * @throws JsonException
+     */
+    public static function format(
+        string $json,
+        array $arrayKeys = [],
+        int $indent = self::DEFAULT_INDENT,
+        bool $formatJson = true,
```

PHPMD (2.15.0)

The method format has a boolean flag argument $formatJson, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)